packages/block-library/package.json

```diff
@@ -29,7 +29,8 @@
 		"src/**/*.scss",
 		"src/navigation-link/index.js",
 		"src/template-part/index.js",
-		"src/query/index.js"
+		"src/query/index.js",
+		"src/quote/v2/index.js"
```

Comment:
Curious about this change, how do we decide to add a file here.

Reply:
My understanding is that we use this to signal webpack not to remove things that aren't part of the export in those files. In theory, it should remove the `addFilter` call. In practice, though, I tested without adding this and things worked as expected. Not sure if there's some webpack config missing, but wanted to play it safe and avoid the situation in which the v2 block stops working because of some webpack config changed.